packages/core/.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "parserOptions": {
+        "project": ["./tsconfig.*?.json"]
+      },
+      "rules": {}
+    },
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "import/no-extraneous-dependencies": "off"
```

Comment:
We should turn off this rule because we have hoisted dependencies and in the closest `package.json` we do not have dependencies. 
Here is a documentation link:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

Reply:
@RomanReznichenko to make it works as expected you just need to specify `packageDir` for `no-extraneous-dependencies.md`

```js
"import/no-extraneous-dependencies": [
  "error",
  {
    "packageDir": ".",
    "devDependencies": false,
    "optionalDependencies": false,
    "peerDependencies": false
  }
],

// ...

"import/no-extraneous-dependencies": [
  "error",
  {
    "packageDir": ".",
    "devDependencies": true
  }
]
```